### DISCUSSION TITLE
Pinia and monorepos

### DISCUSSION THREAD

**Discussion by carelledjuffo** [vuejs/pinia#1296] [Help and Questions] [answered]
My Company just migrated to Pinia lately and we wanted to implement a monorepo codebased structure.

How does Pinia work with monorepos and npm workspaces.
 Is it possible to use a store that was declared in Vue Project A, in another Project B found in the same monorepo?

I tried during this but it never worked, so I was wandering if this is possible in pinia and how can I proceed.

**Comment by posva** [comment, marked as answer]
It's possible as long as only one copy of Vue and pinia are used in the project and one single pinia instance is shared across projects
